distro/scripts/startup.sh

```diff
@@ -1,11 +1,17 @@
 #! /bin/sh
 
 LOG_PATH="/var/log/wsl-vpnkit.log"
-USERPROFILE=$(wslpath "$(powershell.exe -c 'Write-Host -NoNewline $env:USERPROFILE')")
+USERPROFILE=$(wslpath "$(powershell.exe -NoLogo -NoProfile -c 'Write-Host -NoNewline $env:USERPROFILE')")
 VERSION="$(cat /app/version)"
+CONF_PATH="$USERPROFILE/wsl-vpnkit/wsl-vpnkit.conf"
 
 touch $LOG_PATH
 
+mkdir -p "$USERPROFILE/wsl-vpnkit/"
+if [ ! -f "$CONF_PATH" ]; then
+  cp /app/defaults.conf "$CONF_PATH"
+fi
```

Comment:
To minimize the changes automatically made by the script to the user's machine, I would rather this to be added as part of the startup text so users could manually create the config file. That will remove the need to care about migrating the config if something ever changes.

Reply:
Agree. The main motivation was to separate the evaluated variables from the constants.